README.md

```diff
@@ -70,6 +70,68 @@ DeclareSchema::Migration::Migrator.before_generating_migration do
 end
 ```
 
+## Declaring Character Set and Collation
+_Note: This feature currently only works for MySQL database configurations._
+
+With the addition of emojis, there has come a need to dynamically define the
+character set and collation for individual tables and columns in the database.
+With `declare_schema` this can be configured at three separate levels
+
+### Global Configuration
+The character set and collation for all tables and fields can be set at the global level
+using the `Generators::DeclareSchema::Migrator.default_charset=` and
+`Generators::DeclareSchema::Migrator.default_collation=` configuration methods.
+
+For example, adding the following to your `config/initializers` directory will
+turn all tables into emoji supporting tables:
```

Comment:
Let's stick with calling it "utf8mb4" rather than "emoji supporting". This applies below too.

Reply:
Done 556ad3e16ab8af63e5f0da68975a77712a3c2f6e